src/main/kotlin/com/hypto/iam/server/service/SsoLoginSevice.kt

```diff
@@ -1,28 +1,61 @@
 package com.hypto.iam.server.service
 
 import com.hypto.iam.server.configs.AppConfig
+import com.hypto.iam.server.exceptions.EntityNotFoundException
+import com.hypto.iam.server.exceptions.UnknownException
 import com.hypto.iam.server.models.AuthUrlResponse
 import com.hypto.iam.server.models.SsoLoginRequest
+import com.hypto.iam.server.security.AuthenticationException
 import com.workos.WorkOS
+import com.workos.common.exceptions.BadRequestException
+import com.workos.common.exceptions.NotFoundException
+import com.workos.common.exceptions.UnauthorizedException
```

Comment:
workOS exception names may be confused with our own exception names. Can we prefix all workOS exception imports with the term WorkOS?

```suggestion
import com.workos.common.exceptions.BadRequestException as WorkOSBadRequestException
import com.workos.common.exceptions.NotFoundException as WorkOSNotFoundException
import com.workos.common.exceptions.UnauthorizedException as WorkOSUnauthorizedException
```

Reply:
Renamed